client-tools/Dump/arangodump.cpp

```diff
@@ -51,9 +53,54 @@
 using namespace arangodb;
 using namespace arangodb::application_features;
 
+class ArangoDumpInitializer {
```

Comment:
Is this class only declared but not used anywhere? I can't find any usages of it.